packages/strapi-plugin-upload/admin/src/components/HomePageContent/index.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
```

Comment:
If this component is only used once can you put it inside the HomePage container so we can access it more easily?